get-started/02_our_app.md

```diff
@@ -35,12 +35,40 @@ we have created a ZIP file containing the application.
 
 ## Build the app's container image
 
-In order to build the application, we need to use a `Dockerfile`. A
-Dockerfile is simply a text-based script of instructions that is used to
-create a container image. If you've created Dockerfiles before, you might
-see a few flaws in the Dockerfile below. But, don't worry. We'll go over them.
+In order to build the application, you'll need to use a `Dockerfile`. A
+Dockerfile is simply a text-based file with no file extension. A Dockerfile contains a script of instructions that are used to create a container image. 
 
-1. In the `app` folder, the same location as the `package.json` file, create a file named `Dockerfile` with the following contents.
+1. In the `app` folder, the same location as the `package.json` file, create a file named `Dockerfile`. You can use the following commands to create a Dockerfile based on your operating system.
+
+   <ul class="nav nav-tabs">
+     <li class="active"><a data-toggle="tab" data-target="#mac-linux">Mac / Linux</a></li>
+     <li><a data-toggle="tab" data-target="#windows">Windows</a></li>
+   </ul>
+   <div class="tab-content">
+   <div id="mac-linux" class="tab-pane fade in active" markdown="1">
+
+    In the terminal, run the following commands.
+
+   ```console
+   $ cd /path/to/app
+   $ touch Dockerfile
+   ```
+
+   <hr>
+   </div>
+   <div id="windows" class="tab-pane fade" markdown="1">
+
+   In the Windows Command Prompt, run the following commands.
```

Comment:
```suggestion
   In the Windows Command Prompt, run the following commands listed below.
```

Reply:
Thanks jerae-duffin. Updated in the following commit.